src/consensus/softforks.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015 Eric Lombrozo
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SOFTFORKS_H
+#define BITCOIN_SOFTFORKS_H
+
+#include <cstddef>
+
+class CBlockIndex;
+namespace Consensus {
+
+struct Params;
+namespace VersionBits { class BlockRuleIndex; }
+
+namespace SoftForks {
+
+enum Rule
```

Comment:
Can you move this enum to consensus/params.h ?
It will be eventually necessary to expose the versionbits functionality to libconsensus.